docs/ops.html

```diff
@@ -24,6 +24,8 @@
 <h4><a id="basic_ops_add_topic" href="#basic_ops_add_topic">Adding and removing topics</a></h4>
 
 You have the option of either adding topics manually or having them be created automatically when data is first published to a non-existent topic. If topics are auto-created then you may want to tune the default <a href="#topic-config">topic configurations</a> used for auto-created topics.
+If automatic creation of topics is enabled, whenever a broker receives a request for unknown topic, it creates the topic. During this period when broker is creating the topic, any produce request sent to it can timeout and potentially lead to message loss. With proper configuration of retries and retry.backoff.ms, this can be avoided.
```

Comment:
It seems this description is not quite accurate. Topic creation is only on TopicMetadataRequest, not any request. We should also mention max.block.ms. And I don't think this will cause potential data loss - we never claimed the message was successfully sent.

Based on the discussion on the ticket, personally I think it is more important that the error message in the exceptions themselves convey the right information.  Do you think it would help if we improve the error message in the TimeoutException thrown to the user due to metadata timeout? Because the exceptions in the sender thread will be accompanied with retrying logging entry, it seems we are fine there?